### PR TITLE
Restores old monkey knockdowns (monkey nerf)

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Monkey knockdown used to be on Crossed until it was made an AI here: https://github.com/tgstation/tgstation/pull/55238

Now it instead uses COMSIG_ATOM_ENTERED, which only takes into account walking over the monkey. Something that you can't do unless you are non-dense or they are lying on the floor, in which case you aren't even knocking them down?
Now it uses COMSIG_MOVABLE_CROSS instead so it's how it used to be, actually functional and useful.

## Why It's Good For The Game

Restores a lost feature to counter monkeys, knocking them over.

## Changelog

:cl:
fix: you can knock monkeys over by crossing them again.
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
This just makes https://github.com/tgstation/tgstation/pull/65040 completely irrelevant which makes me feel like I'm cheating GBP out of doing this only 1 hour after it was merged, so I would be fine if this gets GBP no update.

**Comment by ATH1909** [comment]
are we sure we want this back? it always seemed weird and janky to me, and doesn't it kind of make monkey swarms irrelevant? monkey swarms are cool.

**Comment by JohnFulpWillard** [comment]
> are we sure we want this back? it always seemed weird and janky to me, and doesn't it kind of make monkey swarms irrelevant? monkey swarms are cool.

Make a balance PR if you want to remove it but I'm just fixing it.

File: code/datums/ai/monkey/monkey_controller.dm [hunk 1]
```diff
@@ -58,25 +56,37 @@ have ways of interacting with a specific mob and control it.
...
 // Stops sentient monkeys from being knocked over like weak dunces.
 /datum/ai_controller/monkey/on_sentience_gained()
 	. = ..()
-	qdel(GetComponent(/datum/component/connect_loc_behalf))
+	UnregisterSignal(pawn, COMSIG_MOVABLE_CROSS)
 
 /datum/ai_controller/monkey/on_sentience_lost()
 	. = ..()
-	AddComponent(/datum/component/connect_loc_behalf, pawn, loc_connections)
+	RegisterSignal(pawn, COMSIG_MOVABLE_CROSS, .proc/on_crossed)
```
Comment (tralezab) [inline code/datums/ai/monkey/monkey_controller.dm:88]: I don't really get why this is on AI but I realize you're fixing this not making changes, so its fine

**Review by tralezab — APPROVED** [review]
accidentally told you the one problem you had over discord haha so i guess we'll let the tests run

**Pull request merged**